src/main/java/frc/team2412/robot/commands/intake/AllInCommand.java

```diff
@@ -20,16 +25,98 @@ public void initialize() {
 
 	@Override
 	public void execute() {
+		// intake rejecting
+		if (intakeSubsystem.intakeFrontSeesNote()) {
+			if (!intakeSubsystem.getRejectOverride()) {
+				intakeSubsystem.intakeBackReject();
+				intakeSubsystem.intakeLeftReject();
+				intakeSubsystem.intakeRightReject();
+			} else {
+				intakeSubsystem.intakeBackStop();
+				intakeSubsystem.intakeLeftStop();
+				intakeSubsystem.intakeRightStop();
+			}
+
+			if (controls != null) {
+				Commands.race(new RumbleCommand(controls), new WaitCommand(1)).schedule();
+			}
+		}
+
+		if (intakeSubsystem.intakeBackSeesNote()) {
+			if (!intakeSubsystem.getRejectOverride()) {
+				intakeSubsystem.intakeFrontReject();
+				intakeSubsystem.intakeLeftReject();
+				intakeSubsystem.intakeRightReject();
+			} else {
+				intakeSubsystem.intakeFrontStop();
+				intakeSubsystem.intakeLeftStop();
+				intakeSubsystem.intakeRightStop();
+			}
+
+			if (controls != null) {
+				Commands.race(new RumbleCommand(controls), new WaitCommand(2)).schedule();
```

Comment:
Note that this will cause a new rumble command to be scheduled every ~20 ms until the note stops triggering the sensor, which would be a bit excessive (and might lead to a crash depending on the maximum number of commands).

Reply:
This should be fixed now